dockerfiles/base/tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
yes, we do, the bats used by launcher does not have docker inside, thus tests for launcher were failing where docker was involved. Problem of bats images is that they don't have docker. Another problem is that those images are not lightweight. On the contrary this image I used is base on eclipse/che-base which is based on alpine so it is really small image and has almost everything we need for tests.